app/models/previously_published_item.rb

```diff
@@ -27,6 +27,14 @@ def first_published_at
     previously_published_item.first_published_at
   end
 
+  def set_last_edited_at?
```

Comment:
Yup I agree. Any methods with get/set prefixes get me twitchy. I'll see if I can change both of them without the application burning.